src/App.css

```diff
@@ -3,7 +3,6 @@ body {
   display: flex;
   align-items: center;
   justify-content: center;
-  padding: 15%;
```

Comment:
So what I meant in my previous comment is that you should consolidate these two body styles into 1 set of styles for it in just 1 file. Can you do that please?